src/common/strong_typedef.cpp

```diff
@@ -64,7 +64,5 @@ template class StrongTypeAlias<noisepage::selfdriving::pilot::tags::tree_node_id
 template class StrongTypeAlias<noisepage::storage::tags::col_id_t_typedef_tag, uint16_t>;
 template class StrongTypeAlias<noisepage::storage::tags::layout_version_t_typedef_tag, uint16_t>;
 template class StrongTypeAlias<noisepage::transaction::tags::timestamp_t_typedef_tag, uint64_t>;
-// template class StrongTypeAlias<noisepage::type::tags::date_t_typedef_tag, uint32_t>;
```

Comment:
These were commented out in #1628 but I forgot to go back and actually delete the lines.